carbon-design-system/carbon
# fix: add autoalign in button

Closes #17168 

#### Changelog

**New**

Added autoAlign in button component for IconButton's tooltip

#### Testing / Reviewing

1. Go to Storybook > Button > Icon Button
2. change control of `autoAlign` to true
3. Tooltip in IconButton should place as per space available.
